doc/ChangeLog.md

```diff
@@ -9,15 +9,36 @@ All notable changes to the project are documented in this file.
 
 ### Changes
 - Initial RISC-V (riscv64) support: StarFive VisionFive2
+- Massive updates to the NanoPi R2S:
+  - Update Linux kernel to v6.10.3 and sync defconfig with aarch64
```

Comment:
I do not really like this, i think all platform should use the same kernel (if possible). To ease maintainability for potental bugfixes etc. To get But that is just my principles, please challenge it.

Reply:
Interesting point of discussion.  I think of it more as a long-term goal to work towards rather than an absolute.  In particular to keep differences between Infix-based boards to a minimum, but unlike previous places of employment I'm not as conservative now.

I see both the NanoPi and the VisionFive boards as references for Infix, to help lower the barrier of entry, ease adoption, and show off the operating system's capabilities.  In that sense some boards will have features that it doesn't share with others, even though they share the same base architecture.  E.g., with/without: switchcore, wifi, cellular, etc.

It's also a question of the number of users on a board, paying customers vs freebie users.  I don't think we should spend too much time on maintaining and testing support for all boards in a common kernel, for instance.  I tried that with the EspressoBIN and MacchiatoBIN boards, but the pace at which our development progressed on the `aarch64_defconfig` it was impossible to keep them working -- same with the CRB board.  I've actually planned to break out more boards from the standard aarch64 build to be able to maintain them more easily, because man power is another component we need to consider here.

So, that's where I'm coming from.  Love to hear what @wkz has to say on this as well.

Reply:
What i mean is if we use the same kernel for the "platform" we could for example build a small test system with R2S (or proffered,  something with 2 more ports) and actually gain from it.

And if a (non-maintained) board breaks after an upgrade, so be it, open a bug report. But at least it has the same patch level as all other supported boards and we can draw conclusions about behavior.

Reply:
OK, we've had a talk about this AFK now too.  Can we agree on merging this for now, and postpone the merge to the `aarch64_defconfig` in the next PR, @lazzer + @wkz ?

Then I could base my planned refresh of the EspressoBIN support on this and coax it all together in the same build for that PR.  I'd like to get on that as soon as possible, even this coming weekend :nerd_face:

Reply:
Thats fine for me.